envs/quadrotors/tests/test_quads.py

```diff
@@ -34,20 +57,13 @@ def test_quad_multi_env(self):
         env_name = 'quadrotor_multi'
         cfg = default_cfg(env=env_name)
         self.assertIsNotNone(create_env(env_name, cfg=cfg))
+        run_multi_quadrotor_env(env_name, cfg)
 
-        env = create_env(env_name, cfg=cfg)
-        env.reset()
-        for i in range(100):
-            obs, r, d, info = env.step([env.action_space.sample() for _ in range(env.num_agents)])
-
-        n_frames = 5000
-        env = create_env(env_name, cfg=cfg)
-        env.reset()
-
-        timing = Timing()
-        with timing.timeit('step'):
-            for i in range(n_frames):
-                obs, r, d, info = env.step([env.action_space.sample() for _ in range(env.num_agents)])
-
-        log.debug('Time %s, FPS %.1f', timing, n_frames * env.num_agents / timing.step)
-        env.close()
+    @unittest.skipUnless(quadrotors_available(), 'quadrotor env package is not installed')
+    @unittest.skipUnless(numba_available(), 'Numba is not installed')
+    def test_quad_multi_env_with_numba(self):
```

Comment:
This won't run in Travis, right? Because quadrotors_available check will fail.

I'm fine with keeping this here, although it'd be better if the tests that we already have in quadrotor env were executed in Travis

Reply:
Maybe consider adding a Travis config to Artem's repo also?

Reply:
I'll ask him to enable and perhaps enable it for that

Reply:
Oh I just saw that Travis does not work for private repositories